styles/theme.css

```
@@ -0,0 +1,51 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@layer utilities {
+  display-1-bold {
+    @apply font-size: 32px line-height:40px font-weight: 700;
+  }
+  display-2-bold {
+    @apply font-size: 24px line-height:32px font-weight: 700;
+  }
+  h1-medium {
+    @apply font-size: 20px line-height:28px font-weight: 500;
+  }
+  h2-medium {
+    @apply font-size: 16px line-height:26px font-weight: 500;
```

```suggestion
    @apply font-size: 16px line-height:24px font-weight: 500;
```
Small fix. I believe our guide wants us to have 24px line height here

---

noted, github copilot was gving me fill in suggestions but I thought I had double checked them against the guide, i do tend to miss tiny things here and there, so I should probably work on that since they usually send me on wild goose chases to find a bug caused by a small typo.  Thanks for catching it!.  The line-height I swear said 12.1 (which was weird to me) but that's not a big deal either... going to game out what im working on today and ill speak with you later 👍